packages/vapor/scss/redesign/colors.scss

```diff
@@ -227,11 +227,11 @@
     --success-40: #6edbc1;
     --success-20: #ccf9ee;
 
-    --warning-100: #ad4f07;
-    --warning-80: #c25d0a;
-    --warning-60: #e8700c;
-    --warning-40: #ffa666;
-    --warning-20: #ffe0cc;
+    --warning-100: #805700;
+    --warning-80: #bf7500;
+    --warning-60: #f0a702;
+    --warning-40: #fdcd66;
+    --warning-20: #ffefcc;
```

Comment:
Are those changes related to the banner style update?

Reply:
No, I took those colors from the figma file, it seemed that the warning color were not the same so I changed them

Reply:
@guscoveo I'm probably a bit outdated, are we using Figma as standard now? Could you please confirm these changes for the palette? Thx!

Reply:
@rosalie-liu Hi Rosalie :) Yeah, we're transitioning from Sketch/Abstract to Figma. But the colours that you see are from the same old library started by Kate/Atecna within Figma (and used within our Sketch initial mocks). In any case and to confirm: yes, Figma will be our source of truth :)